controllers/api.controllers.js

```diff
@@ -10,11 +11,28 @@ const getTopics = (req, res, next) => {
 const getApiEndpoints = (req, res) => {
     selectApiEndPoints(req.query)
     .then((endpoints) => {
-        console.log(endpoints)
         res.status(200).send({endpoints})
     })
 }
 
+const getArticleById = (req, res) => {
+    let id = req.params.article_id
+    selectArticleById(id)
+    .then((rows) => {
+        if (rows.length === 0) {
+            res.status(404).send({msg: "Not found"})
```

Comment:
This should be caught in the model, preferably using Promise.reject to send the error. Doing it this way may need refactoring in the future 👍